externs/browser/html5.js

```diff
@@ -3845,6 +3845,11 @@ Window.prototype.WebKitMutationObserver;
  */
 Window.prototype.MozMutationObserver;
 
+/**
+ * @type {Document}
```

Comment:
Why give it a different type from "var document"?
https://github.com/google/closure-compiler/blob/master/externs/browser/window.js#L44

Also, why put this in html5.js, rather than say window.js?